<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dictionary</class>
 <widget class="QWidget" name="dictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AKA多功能词典</string>
  </property>
  <property name="toolTip">
   <string>发音</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>请输入单词</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="WordIput"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>image/search.png</normaloff>image/search.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="toolTip">
              <string>保存</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>image/save.png</normaloff>image/save.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="toolTip">
              <string>全文翻译</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>image/fulltext.png</normaloff>image/fulltext.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="toolTip">
              <string>设置</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>image/option.png</normaloff>image/option.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3"/>
           </item>
           <item>
            <widget class="QPushButton" name="soundbt">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>image/laba.png</normaloff>image/laba.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="toolTip">
              <string>添加词库</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>image/ciku.png</normaloff>image/ciku.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="handw_input">
             <property name="toolTip">
              <string>手写输入</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>image/handw.png</normaloff>image/handw.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTextBrowser" name="MeanBrowser"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
